app/renderer/components/navigation/navigationBar.js

```diff
@@ -180,12 +263,13 @@ class NavigationBar extends React.Component {
       return null
     }
 
-    return <div id='navigator'
+    return <div id='navigationBar'
       ref='navigator'
+      data-test-id='navigationBar'
       data-frame-key={this.props.activeFrameKey}
       className={cx({
         titleMode: this.props.titleMode,
-        [css(styles.navigator_wide)]: this.props.isWideURLbarEnabled
+        [css(styles.navigationBar, (this.props.isDarwin && this.props.isFullScreen) && styles.navigationBar_isDarwin_isFullScreen, this.props.titleMode && styles.navigationBar_titleMode, this.props.isWideURLbarEnabled && styles.navigationBar_wide)]: true
```

Comment:
I would suggest moving this to Aphrodite to avoid this long line but titleMode needs its own refactor.